src/lib.rs

```diff
@@ -397,8 +400,45 @@ macro_rules! redis_json_module_create {(
             pre_command_function: $pre_command_function_expr,
         }
 
+        pub fn check_minimal_version_for_short_read(ctx: &Context) -> bool {
+            // Minimal versions: 6.2.5
```

Comment:
json do not have aux fields so you can support short read even before this version.

Reply:
@MeirShpilraien So this would make `get_redis_version` in `redismodule-rs` unnecessary, right?